community-operators/rook/rook.v0.1.0.csv.k8s.yaml

```diff
@@ -0,0 +1,306 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  creationTimestamp: null
+  name: rook.v0.1.0
+  namespace: placeholder
+  annotations:
```

Comment:
Additional annotations can be found here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md